app/src/main/AndroidManifest.xml

```diff
@@ -19,6 +20,13 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+
+        <provider
+            android:name="androidx.work.impl.WorkManagerInitializer"
```

Comment:
use
android:authorities="${applicationId}.workmanager-init"
gradle with handle

Reply:
Don't know what "gradle with handle" means but first part I've done 👌

Reply:
*
with -> will

just copy paste line:
android:authorities="${applicationId}.workmanager-init"
instead of:
android:authorities="com.github.mik629.android.fundamentals.workmanager-init"

Reply:
Ah, I assumed so but hadn't understood that you were talking about `applicationId`. Thanks!